generators/java/reactor-netty/generator/src/main/kotlin/io/github/fomin/oasgen/java/reactor/netty/ReactorNettyClientWriter.kt

```diff
@@ -93,10 +93,29 @@ class ReactorNettyClientWriter(
                     }
                 }.joinToString("")
 
-                val parameterStrDeclarations = javaOperation.parameters.mapIndexed { index, javaParameter ->
-                    val stringWriteExpression =
-                            converterRegistry[javaParameter.schemaParameter.schema()].stringWriteExpression("param$index")
-                    "String param${index}Str = param$index != null ? $stringWriteExpression : null;"
+                val parameterStrDeclarations = javaOperation.parameters.mapIndexedNotNull { index, javaParameter ->
+                            if (javaParameter.schemaParameter.parameterIn != ParameterIn.HEADER) {
+                                val stringWriteExpression =
+                                        converterRegistry[javaParameter.schemaParameter.schema()].stringWriteExpression("param$index")
+                                "String param${index}Str = param$index != null ? $stringWriteExpression : null;"
+                            } else {
+                                null
+                            }
+                }
+
+                val headerArgs = javaOperation.parameters
+                        .mapIndexedNotNull { index, parameter ->
+                            if (parameter.schemaParameter.parameterIn == ParameterIn.HEADER) {
+                                val stringWriteExpression =
+                                        converterRegistry[parameter.schemaParameter.schema()].stringWriteExpression("param$index")
+                                """.add("${parameter.name}", param$index != null ? $stringWriteExpression : null)"""
+                            } else {
+                                null
+                            }
+                        }
+                val headerArgsBlock = when(headerArgs.isNotEmpty()) {
+                    true -> """|        .headers(headers -> headers${headerArgs.indentWithMargin(5)})""".trimMargin()
```

Comment:
strange formatting...
margin does not make sense for single line literals.

why `indentWithMargin(5)` is used here?

Reply:
> strange formatting...
> margin does not make sense for single line literals.
> 
> why `indentWithMargin(5)` is used here?

I am not sure... but I can't do that several parameters of headers have the same gaps

Reply:
> strange formatting...
> margin does not make sense for single line literals.
> 
> why `indentWithMargin(5)` is used here?

Added a little change of format. indentWithMargin(5) was added to move internal block to right